usmqe_tests/api/gluster/test_volume.py

```diff
@@ -19,6 +20,108 @@
 """
 
 
+def test_create_brick_valid(
+        valid_cluster_id,
+        valid_brick_path,
+        valid_session_credentials):
+    """@pylatest api/gluster.create_brick_valid
+        API-gluster: create_brick
+        ******************************
+
+        .. test_metadata:: author fbalak@redhat.com
+
+        Description
+        ===========
+
+        Create ``GlusterCreateBrick`` job with a ``valid_brick_path`` with nodes from cluster
+        with ``valid_cluster_id``
+
+        .. test_step:: 1
+
+                Connect to Tendrl API via POST request to ``APIURL/:cluster_id/GlusterCreateBrick``
+                Where cluster_id is set to predefined value.
+
+        .. test_result:: 1
+
+                Server should return response in JSON format:
+
+                Return code should be **202** with data ``{"message": "Accepted"}``.
+                job should finish.
+                """
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+
+    cluster_info = [x for x in api.get_cluster_list()
+                    if x["integration_id"] == valid_cluster_id]
+    nodes = cluster_info[0]["nodes"]
+
+    job_id = api.create_bricks(
+        valid_cluster_id,
+        nodes,
+        valid_brick_path)["job_id"]
+    api.wait_for_job_status(job_id)
+    """@pylatest api/gluster.create_brick_valid
+        API-gluster: create_brick
+        ******************************
+
+        .. test_metadata:: author fbalak@redhat.com
+
+        Description
+        ===========
+
+        Check if the bricks were created on hosts of cluster with ``valid_cluster_id``.
+
+        .. test_step:: 2
+
+                Via ssh check on cluster nodes that there exists
+                directory with ``valid_brick_path``:
+                    [ -d ``valid_brick_path`` ] && echo "exists"
+
+        .. test_result:: 2
+
+                There should be string ``exists`` in output of ssh.
+                """
+    SSH = usmssh.get_ssh()
+    pytest.check(
+        len(nodes) > 0,
+        "In cluster have to be at least one node. There are {}".format(len(nodes)))
+    cmd_exists = "[ -d {} ] && echo 'exists'".format(valid_brick_path)
+    cmd_fs = 'df -T "{}" | awk \'{{print $2}}\' | tail -n1'.format(valid_brick_path)
```

Comment:
I consider this to be just an initial version (based on our discussion last week), I'm ok with it as long as you add a todo note next to it.